apps/zbugs/src/hooks/use-element-size.ts

```diff
@@ -1,12 +1,12 @@
 import {useLayoutEffect, useState} from 'react';
 
-export function useElementSize(elm: HTMLElement | null) {
+export function useElementSize(elm: React.RefObject<HTMLElement>) {
```

Comment:
this was the value of `ref.current` before which would always be stale when the `useLayoutEffect` callback was called.

It just so happened to work before because we'd always do two renders via `useQuery`. The second one would pass a new `elm` in.